extension/js/common/core/pgp-msg.ts

```diff
@@ -230,6 +230,8 @@ export class PgpMsg {
   }
 
   public static diagnosePubkeys: PgpMsgMethod.DiagnosePubkeys = async ({ privateKis, message }) => {
+    // openpgp is undefined here
```

Comment:
@tomholub I feel stupid on this task :laughing: 

Where should I import OpenPGP.js so it'll be available here?

Reply:
Any place that may be the parent of a `pgp_block`:
1) in the content script
2) in encrypted inbox page

But both should already have it:
content script:
![image](https://user-images.githubusercontent.com/6306961/73491601-f3814e00-43e9-11ea-9eab-160b526d53f5.png)

inbox:
![image](https://user-images.githubusercontent.com/6306961/73491685-1f043880-43ea-11ea-935e-f2b84d484798.png)

What is the stack trace of the debugger?

Reply:
```
Error: BrowserMsg() sendRawResponse::TypeError: Cannot read property 'message' of undefined
    at Function.BrowserMsg.jsonToErr (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/js/common/browser/browser-msg.js:318:15)
    at processRawMsgResponse (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/js/common/browser/browser-msg.js:245:35)

[callerStack]
    at chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/js/common/browser/browser-msg.js:217:120
    at new Promise (<anonymous>)
    at Function.BrowserMsg.sendAwait (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/js/common/browser/browser-msg.js:215:18)
    at Object.pgpMsgDiagnosePubkeys (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/js/common/browser/browser-msg.js:69:57)
    at PgpBlockViewErrorModule.handlePrivateKeyMismatch (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/chrome/elements/pgp_block_modules/pgp-block-error-module.js:28:62)
    at async PgpBlockViewDecryptModule.decryptAndRender (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/chrome/elements/pgp_block_modules/pgp-block-decrypt-module.js:129:29)
    at async PgpBlockViewDecryptModule.initialize (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/chrome/elements/pgp_block_modules/pgp-block-decrypt-module.js:63:25)
    at async PgpBlockView.render (chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/chrome/elements/pgp_block.js:26:17)

url: chrome-extension://dfbpklhlbpikpllkojnfhdhgbomoponf/chrome/elements/pgp_block.htm?frameId=frame_OLkOOTxzNo&message=&hasPassword=___cu_false___&msgId=16f29cf85aa2cd61&senderEmail=mindoutofframe%40gmail.com&isOutgoing=___cu_false___&acctEmail=flowcrypt.compatibility%40gmail.com&parentTabId=17%3A0
```

Reply:
I'll take a look, this should have worked.

Reply:
I fixed it. There was a long-present issue with loading openpgp.js in content scripts. If you look into `flowcrypt-browser/build/chrome-consumer/js/content_scripts/webmail_bundle.js`, you'll notice that webmail bundle is really just concatenated source files with import statements commented out (poor man's bundler).

Once concatenated, this happened:

```js
// ...
const requireOpenpgp = () => {
   // ...
   return openpgp; // where I really meant global window.openpgp
}
// ...
const openpgp = requireOpenpgp();
```
The `return openpgp;` would fail because it thought I'm referring to the const that's defined below, where I actually meant the global one.

Did not not happen outside of content scripts, because in other pages `const openpgp = ` is in a separate file/module.